src/registrar/templates/application_org_election.html

```diff
@@ -6,7 +6,11 @@ <h2 class="margin-bottom-05">
     Is your organization an election office?  <abbr class="usa-hint usa-hint--required" title="required">*</abbr>
   </h2>
 
-  <p>Answer “yes” if the primary purpose of your organization is to manage elections.</p>
+  <p>An election office is a government entity whose <i>primary</i> responsibility is overseeing elections and/or conducting voter registration.
```

Comment:
*comment (non-blocking)*: I think that the semantic `<em>...</em>` is preferred for emphasis markup in the WCAG standards https://www.w3.org/WAI/WCAG21/Techniques/html/H49